app/views/users/index.html.erb

```diff
@@ -8,6 +8,7 @@
 	     <p><%= user.name %></p>
 			 <p> <%= image_tag user_avatar(user) , class: "img-fluid med-profile-img" %> </p>
 			 <%= link_to "See Profile", user, :class => "bg-red font-white p-1 profile" %>
+			 <%= button_to "Remove Friend",{:controller => 'friendships', :action => 'destroy', :id => friendship.id }, {:method => :patch, :class => "bg-red font-white p-1 acceptingfriend"} %>
```

Comment:
We don't have destroy action in the friendship controller.
Let's add it and use `:method => :patch`.
This is the reason reject friend button is confirming a Friendship record instead of deleting it.